### PR TITLE
Fixed More OpCodes

### PR DESCRIPTION
**Pull request by Diamantino-Op**

### CODE REVIEW

File: tests/IL2CPU.Compiler.Tests/ILStackAnalysisTests.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 namespace IL2CPU.Compiler.Tests
 {
-    public class ExampleMethods
```
Comment (quajak) [inline tests/IL2CPU.Compiler.Tests/ILStackAnalysisTests.cs:14]: Please undo the unsafe related changes

File: source/Cosmos.IL2CPU/ILOpCodes/OpNone.cs [hunk 2]
```diff
@@ -684,6 +685,11 @@ public override void DoInterpretStackTypes()
             StackPushTypes[0] = xTypeShift;
             return;
           }
+          if (xTypeValue == typeof(UIntPtr) && (xTypeShift == typeof(int) || xTypeShift == typeof(uint)))
```
Comment (quajak) [inline source/Cosmos.IL2CPU/ILOpCodes/OpNone.cs:688]: Please include the case `xTypeValue == typeof(IntPtr)`

Comment (Diamantino-Op) [reply source/Cosmos.IL2CPU/ILOpCodes/OpNone.cs:688]: There is already

i just copied that if and pasted down a line and changed that to UIntPtr

Comment (Diamantino-Op) [reply source/Cosmos.IL2CPU/ILOpCodes/OpNone.cs:688]: the one you see deleted below, was a duplicate of the one above it

**Review by quajak — APPROVED** [review]

**Pull request merged**